client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button.jsx

```diff
@@ -33,15 +33,15 @@ export default React.createClass( {
 
 	render() {
 		const { site, redirect, linkDisplay } = this.props;
-		const buttonElement = linkDisplay ? 'button' : Button;
 
 		const buttonClasses = classNames( {
 			button: true,
 			'disconnect-jetpack-button': true,
 			'is-link': linkDisplay
 		} );
 
-		const buttonProps = assign( {}, this.props, {
+		const omitProps = [ 'site', 'redirect', 'isMock', 'linkDisplay' ];
+		const buttonProps = assign( {}, ...omit( this.props, omitProps ), {
```

Comment:
This can be shortened to:
```javascript
const buttonProps = { ...omit( this.props, omitProps ), id:`disconnect-jetpack-${ site.ID }`, className: buttonClasses, compact: true };
```